skunk/rust/skunk/USAGE

```diff
@@ -0,0 +1,3 @@
+                        Runs interpreter on ./test.skunk
+-h | help | --help      Display this help message
+examples file_path      Run the example at file_path
```

Comment:
I think `examples` currently just builds an examples binary, you still need to then run that yourself.

Reply:
Ah yes, whoops, I misunderstood the output

Would renaming this 'build' make as much sense?

Reply:
I also wonder if the default script should be 'main.skunk' rather than 'test.skunk' (so that it works in other projects as well, or is running the tests the best entry point for a skunk project?)

Reply:
Ahh, I realize now that it builds the 'examples' of the file, so the name is perfect.

Reply:
(FYI test.skunk isn't the default entry point, it's just something I did early on before having the examples. Will definitely be replaced once we can figure out what a normal project might look like.)